### DISCUSSION TITLE
Go rewrite status?

### DISCUSSION THREAD

**Discussion by AO12612** [danielmiessler/Fabric#732] [Q&A] [answered]
What is the current timeline for the rewrite and what would be the best way to participate other than github discussions ? (assuming pull requests at this stage would just pollute the process)

**Comment by xssdoctor** [comment, marked as answer]
It's done but super buggy. Still sorting that out

**Reply by sleeper** [reply]
Hey! I guess several people here could help you with that if needed!

**Reply by xssdoctor** [reply]
Emailing you

**Reply by misteral** [reply]
@xssdoctor i can help too

**Reply by AO12612** [reply]
@xssdoctor, as I said, me too.
